packages/cloud/src/routes/tenants.ts

```diff
@@ -0,0 +1,25 @@
+import { Router } from '@withtyped/server';
+import { z } from 'zod';
+
+import { client } from '#src/queries/index.js';
+import { createTenantsQueries } from '#src/queries/tenants.js';
+import { getTenantIdFromManagementApiIndicator } from '#src/utils/tenant.js';
+
+const { getManagementApiLikeIndicatorsForUser } = createTenantsQueries(client);
+
+export const tenants = new Router('/tenants').get(
+  '/',
+  { response: z.object({ id: z.string(), indicator: z.string() }).array() },
+  async (context, next) => {
+    const { rows } = await getManagementApiLikeIndicatorsForUser('some_user_id');
```

Comment:
Is this on purpose? `some_user_id` looks strange here

Reply:
it's for test and on purpose, will be replaced with the result of auth soon